gateway/main/settings_admin_panel.py

```diff
@@ -0,0 +1,41 @@
+"""
+Django settings for admin panel application.
+"""
+
+from .settings_api import *  # pylint: disable=wildcard-import, unused-wildcard-import
```

Comment:
Here you can take a look how we are managing settings from a base configuration.

Reply:
I left in the summary of the PR the reason behind we need to use `api` and not `base`, btw.